fast_matched_filter/src/matched_filter.c

```diff
@@ -318,6 +318,96 @@ void matched_filter_precise_no_sum(
     }
 }
 
+void matched_filter_variable_precise(
```

Comment:
That would be a good idea, but it would require a more consequential rewrite. I admit I don't have the time at the moment, but it's something that Eric and I could potentially work on...?